### ISSUE TITLE
fibers@3.1.1 not compatible with Node 12?

### ISSUE THREAD

**Issue by rsinger86** [pstadler/flightplan#178]
Thanks for this amazingly useful tool - I really love the API. 

I can't get flightplan to work with Node 12 because it appears that flightplan depends on fibers@3.1.1, but that version of fibers is incompatible with Node 12. 

Would it be possible to update to version 4 of fibers?

**Comment by cullylarson** [comment]
I'm having the same problem. Using node 12 on macos, failing to build fibers@3.1.1. Getting errors like this:

```
> fibers@3.1.1 install /Users/me/npm/lib/node_modules/flightplan/node_modules/fibers
> node build.js || nodejs build.js

  CXX(target) Release/obj.target/fibers/src/fibers.o
../src/fibers.cc:68:2: error: no template named 'Handle'
        Handle<String> NewLatin1String(Isolate* isolate, const char* string) {
        ^
../src/fibers.cc:69:10: error: no viable conversion from returned value of type 'Local<v8::String>'
      to function return type 'int'
  ...String::NewFromOneByte(isolate, (const uint8_t*)string, NewStringType::kNormal).ToLocalChecked();

../src/fibers.cc:72:2: error: no template named 'Handle'
        Handle<String> NewLatin1Symbol(Isolate* isolate, const char* string) {
...
```

**Comment by pstadler** [comment]
Glad you like this project. Please provide a PR.

**Comment by Zwimber** [comment]
Checked and the commit seems to work!

**Comment by pstadler** [comment]
This has been fixed with [0.6.20](https://github.com/pstadler/flightplan/releases/tag/0.6.20). Thanks!
